pset-functions.js

```diff
@@ -31,6 +31,19 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+function blackJack(playerCardScore, dealerCardScore ){
+
+  if(playerCardScore > 21 && dealerCardScore > 21){
+    return 0
+  } else if (playerCardScore > dealerCardScore && playerCardScore <= 21 || dealerCardScore > 21){
+    return playerCardScore
+  } else {
+    return dealerCardScore
+  }
+}
+
+console.log(blackJack(19, 22))
+
```

Comment:
@nmikhol Good Job here creating this function to check for the different outcomes to determine the winner in the game. Good job using that `&&` operator to check the two conditions in that statement.
take a look at a solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L30)